tests/parser-inference/jsfmt.spec.js

```diff
@@ -1 +1,48 @@
-run_spec(__dirname);
+const dedent = require("dedent");
+
+run_spec({
+  dirname: __dirname,
+  codes: [
+    {
+      name: "html",
+      filename: "prettier.html",
+      code: dedent`
+        <foo>
+                        <bar/>
+        </foo>
+      `,
```

Comment:
But this looks worse than files IMHO. There is nothing wrong with small files. They should be small.

Reply:
I restored them.